cpp/include/DataStorm/Node.h

```diff
@@ -46,80 +46,6 @@ namespace DataStorm
          */
         Node(Ice::CommunicatorPtr communicator);
 
-        /**
```

Comment:
This constructor and the ones removed below are now unnecessary, because now we can just forward all parameters directly to `Ice::initialize`

Reply:
Why did we add them in the first place? A C++ < 17 limitation?

Reply:
To call `parseCommandLineOptions` from `init` when DataStorm was not in Ice property prefixes.